src/core/utils/EventNormalizer.ts

```diff
@@ -61,12 +63,12 @@ export class EventNormalizer {
     _observer: MutationObserver;
     _selectAllOriginElement: DOMElement; // original selection/target before updating selection
     _mousedownInEditable: MouseEvent; // original mousedown event when starting selection in editable zone
-    _triggerEvent: Function; // callback to trigger on events
+    _sendSignal: (signal: Signal) => void; // callback to trigger on events
```

Comment:
This renaming is funny considering I don't like the term `Signal` and want to think about whether the Normalizer could trigger events in a DOMEvent-compatible format... ;-)